google/cloud/aiplatform/jobs.py

```diff
@@ -2456,25 +2457,30 @@ def update(
         current_job = self.api_client.get_model_deployment_monitoring_job(
```

Comment:
Since, we are calling sync above, which calls get underneath, there is no need to call get here. The self._gca_resource proto can be copied.

Reply:
done